tensorboard/plugins/histogram/BUILD

```diff
@@ -112,7 +112,7 @@ py_library(
 
 py_test(
     name = "summary_test",
-    size = "small",
+    size = "medium",
```

Comment:
Can you add a note about why this needs to be medium?  E.g. `# Ensure enough RAM for test_with_large_counts`

Reply:
Done.